govcd/nsxv_firewall_test.go

```diff
@@ -0,0 +1,136 @@
+// +build edge firewall functional ALL
+
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+func (vcd *TestVCD) Test_NsxvFirewallRule(check *C) {
```

Comment:
I am not fond of tests which tests few methods. Usually test scope is one main function.

Reply:
Could be. I can try to split them into multiple. The only reasons for this is to save on test suite time. (For example to do a TestDelete it would still need to be created)
@dataclouder, @lvirbalas your thoughts?

Reply:
Both approaches are valid and have their own place. However, a composite test is capable of finding deeper and unexpected bugs than a simple test. This test is creating multiple rules and testing the order of them, hence I'm in favor of leaving it as is.